Storj.cyberduckprofile

```diff
@@ -0,0 +1,2558 @@
+<plist version="1.0">
+<dict>
+<key>Protocol</key>
+<string>s3</string>
+<key>Vendor</key>
+<string>Storj Labs</string>
+<key>Description</key>
+<string>Storj DCS</string>
+<key>Default Hostname</key>
+<string>gateway.storjshare.io</string>
+<key>Hostname Configurable</key>
+<true/>
+<key>Default Port</key>
+<string>443</string>
+<key>Port Configurable</key>
+<false/>
+<key>Username Placeholder</key>
+<string>Access Key</string>
+<key>Password Placeholder</key>
+<string>Secret Key</string>
+<key>Authorization</key>
+<string>AWS4HMACSHA256</string>
+<key>Properties</key>
+<array>
+<string>s3.storage.class.options=STANDARD</string>
+<string>s3.upload.multipart.size=67108864</string>
+<string>s3.copy.multipart.size=67108864</string>
+<string>s3.versioning.enable=false</string>
+<string>s3.accelerate.enable=false</string>
+<string>s3.bucket.requesterpays=false</string>
+</array>
```

Comment:
You may also want to restrict the storage class options using `s3.storage.class.options`.

Reply:
Done.  I suspect this doesn't matter on our side, in which case adding it won't hurt.